tests/address-lookup-table-program/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "address-lookup-table",
+  "version": "0.24.0",
```

Nit: other tests use the current version of the repo (`0.30.1`) here, so it would be nice to keep things consistent when possible.
```suggestion
  "version": "0.30.1",
```